cluster/ci/monitoring/mixins/grafana_dashboards/canary.jsonnet

```diff
@@ -0,0 +1,59 @@
+local config =  import '../config.libsonnet';
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local graphPanel = grafana.graphPanel;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+
+local legendConfig = {
+        legend+: {
+            sideWidth: 350
+        },
+    };
+
+dashboard.new(
+        'Release-Informing Jobs Dashboard',
+        time_from='now-2d',
+        schemaVersion=18,
+      )
+.addPanel(
+    (graphPanel.new(
+        'Canary Release Informer States',
+        description='sum(rate(prowjob_state_transitions{job="plank",job_name=~"canary-openshift-ocp-installer-e2e-.*-4.2",job_name!~"rehearse.*",state="success"}[48h]))/sum(rate(prowjob_state_transitions{job="plank",job_name=~"canary-openshift-ocp-installer-e2e-.*-4.2",job_name!~"rehearse.*",state=~"success|failure"}[48h]))',
+        datasource='prometheus',
+        legend_alignAsTable=true,
+        legend_rightSide=true,
+        legend_values=true,
+        legend_current=true,
+        legend_min=true,
+        legend_sort='min',
+        legend_sortDesc=true,
+        min='0',
+        max='1',
+        formatY1='percentunit',
+    ) + legendConfig)
+    .addTarget(prometheus.target(
+        'sum(rate(prowjob_state_transitions{job="plank",job_name=~"canary-openshift-ocp-installer-e2e-aws-fips.*-4.2",job_name!~"rehearse.*",state="success"}[48h]))/sum(rate(prowjob_state_transitions{job="plank",job_name=~"canary-openshift-ocp-installer-e2e-aws-fips.*-4.2",job_name!~"rehearse.*",state=~"success|failure"}[48h]))',
+        legendFormat='canary-openshift-ocp-installer-e2e-aws-fips.*-4.2',
```

Comment:
Could we use more human-readable names for these? For instance, this might be `AWS IPI FIPS`